cmd/minikube/cmd/config/addons_list.go

```diff
@@ -137,6 +137,10 @@ var printAddonsList = func(cc *config.ClusterConfig) {
 }
 
 var printAddonsJSON = func(cc *config.ClusterConfig) {
+	if cc == nil {
```

Comment:
so for JSON we're just returning nothing if there's no cluster?

Reply:
could we not do the same thing we do for table mode? like output the list of addons without a status?

Reply:
Yup, done